web/client/codechecker_client/cmd_line_client.py

```diff
@@ -1322,6 +1322,23 @@ def condition(name, runid, date):
     LOG.info("Done.")
 
 
+def handle_update_run(args):
+
+    init_logger(args.verbose if 'verbose' in args else None)
+
+    client = setup_client(args.product_url)
+
+    run_info = check_run_names(client, [args.run_name])
```

Comment:
When I try to non-existent run, it prints: "The run named 'xyz' was not found. Possible run names are:". After this message no runs are listed from what I could choose, because `check_run_names()` would list only the runs which are also given in its second parameter. I think `check_run_names()` should always list all run names when it gives a hint.

Reply:
I think its not a good idea to list all run names because what if a CodeChecker product contains a huge number of runs like 500. I refactored the code a little bit, please check it.